<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView_Employees">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>931</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QTableView" name="tableView_Inactions">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>931</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_Connections">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>931</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>670</y>
      <width>365</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_addRow">
       <property name="text">
        <string>Добавить строку</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_deleteRow">
       <property name="text">
        <string>Удалить строку</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_update">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>98</width>
      <height>661</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonEmployees_addRow">
        <property name="text">
         <string>Добавить строку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonEmployees_deleteRow">
        <property name="text">
         <string>Удалить строку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonEmployees_update">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonInactions_addRow">
        <property name="text">
         <string>Добавить строку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonInactions_deleteRow">
        <property name="text">
         <string>Удалить строку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonInactions_update">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButtonConnections_addRow">
        <property name="text">
         <string>Добавить строку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonConnections_deleteRow">
        <property name="text">
         <string>Удалить строку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonConnections_update">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
